Products/Zuul/routers/devicemanagement.py

```diff
@@ -72,6 +75,7 @@ def editMaintWindow(self, params):
         """
         facade = self._getFacade()
         data = facade.editMaintWindow(params)
+        audit('UI.MaintenanceWindow.Edit', params['id'])
```

Comment:
for edit operations, do we not include *what* changed in the audit log?  I'm sure I've seen that somewhere...

Reply:
i know that we do that for at least some of them. the way the maintenance window parameters are passed in i'm not sure if there is a good way to specifically glean the changes out but i'll check. good catch.